src/test/java/org/kpmp/repositoryDataset/RepositoryFileDatasetTest.java

```diff
@@ -131,7 +131,28 @@ public void setTissueSource() {
 
     @Test
     public void setReleaseVersion() {
-        repositoryDataset.setReleaseVersion(42);
-        assertEquals(42, repositoryDataset.getReleaseVersion());
+        repositoryDataset.setReleaseVersion("42");
+        assertEquals("42", repositoryDataset.getReleaseVersion());
+        repositoryDataset.setReleaseVersion("Recently Released");
+        assertEquals("Recently Released", repositoryDataset.getReleaseVersion());
     }
+
+    @Test
+    public void setFileNameSort() {
```

Comment:
This should be named something more like 'getFileNameSort' since there is not set method named that.